### ISSUE TITLE
How to change refresh() Header Bearer Token to Bearer Refresh Token ?

### ISSUE THREAD

**Issue by geofany** [sidebase/nuxt-auth#905]
### Describe the feature

How to change refresh() Header Bearer Token to Bearer Refresh Token ?
<img width="674" alt="Screenshot 2024-09-10 at 14 45 47" src="https://github.com/user-attachments/assets/1f15d463-3318-4a73-9c0e-087f4ac8dc6d">
i need to change Header Bearer Token to Bearer Refresh Token 

### Additional information

- [ ] Would you be willing to help implement this feature?

### Provider

- [ ] AuthJS
- [X] Local
- [ ] Refresh
- [ ] New Provider

**Comment by phoenix-ru** [comment]
What exactly do you want?.. `Authorization` header of `refresh` includes the Access token, and Refresh Token should be in the body

**Comment by geofany** [comment]
> What exactly do you want?.. `Authorization` header of `refresh` includes the Access token, and Refresh Token should be in the body

My backend team places the refresh token in the `Authorization` header, instead of in the body.

**Comment by phoenix-ru** [comment]
I don't think we can support custom placements in local provider at the moment. I attempted to create a hook-based data extraction/manipulation for the local provider, but the complexity greatly outweighs advantages gained. Perhaps we'll revisit it later.

At the moment I can't recommend you using the stock module. You can patch it to suit your usecase or create your own authentication
